### ISSUE TITLE
Update Visual Studio Marketplace with 2017 support

### ISSUE THREAD

**Issue by PazerOP** [davemckeown/TidyTabs-VisualStudio#18]
There appears to already be support for VS2017 in this extension, but the Visual Studio Marketplace version still only supports VS2012/2013/2015. https://marketplace.visualstudio.com/items?itemName=davemckeown.TidyTabs

Looks like all that's needed is the latest version compiled and uploaded.

**Comment by vatioz** [comment]
Is this planned or not?

**Comment by vatioz** [comment]
For anyone interested, I created a fork and upgraded it to support VS2017. So if you don't mind downloading the project and building the extension for yourself, here you are: https://github.com/vatioz/TidyTabs-VisualStudio

I also did a pull request with this change: https://github.com/vatioz/TidyTabs-VisualStudio

**Comment by PazerOP** [comment]
@vatioz Your project doesn't compile on a fresh installation of VS2017, lots of missing reference errors which I believe is related to not having previous versions of VS installed. Could you just make a release with the .vsix?

**Comment by AndrueCope** [comment]
Updating it for 2019 would also be good :)

**Comment by ghost** [comment]
> Updating it for 2019 would also be good :)

Your fork doesn't build on a fresh VS2019 either.
Do you have a fix or a pre-built binary?

**Comment by AndrueCope** [comment]
Sadly not. I tried updating it to use the new API but failed :(

**Comment by ghost** [comment]
@AndrueCope In fact I twisted around in NuGet and disabled the unit tests (because it's using an old library) and whatnot, and finally managed to make a build. I am using it in VS2019 now. It seems to work well except that VS complains about the legacy (non-async) loading method.

**Comment by AndrueCope** [comment]
Oh cool! That's gonna make my life better in so many ways. Well done. Here's hoping MS keep sync API going for a while yet. It was that I was trying to work around.

**Comment by ghost** [comment]
So would you like to make a binary release on your forked repo? I had to download Gigabytes of C# development tools just to build this... Hope a release VSIX could help others with zero C#/NuGet/VSIX knowledge like me.

**Comment by AndrueCope** [comment]
Yes, I'll do that. Unfortunately I'm just back at work today after vacation so it'll take a while to get around to it. Hopefully though, later today.

**Comment by ghost** [comment]
That's great. No hurry. Thank you!

**Comment by AndrueCope** [comment]
I've fixed up the solution so that it builds and added a VSIX. What a faff. I suggested to MS that they include this functionality in VS but it didn't get much interest.

**Comment by michaelm7456** [comment]
Hi all,

I've added a Pull Request with some changes to the VSIX Manifest file to allow support for Visual Studio 2019, in case this is of interest.

[#21 - Update VSIX Manifest file to allow Extension to run in Visual Studio 2019.](https://github.com/davemckeown/TidyTabs-VisualStudio/pull/21)

Hope this helps!

**Comment by katerlouis** [comment]
Is there any way to get this extension working in 2022? :(
I just migrated from SublimeText and am really missing this feature.
